pkgs/development/python-modules/macholib/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  unittestCheckHook,
+  altgraph,
+  setuptools,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "macholib";
+  version = "1.16.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ronaldoussoren";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-bTql10Ceny4fBCxnEWz1m1wi03EWMDW9u99IQiWYbnY=";
+  };
+
+  dependencies =
+    [
+      altgraph
+    ]
+    ++ lib.optionals (pythonOlder "3.11") [
+      typing-extensions
+    ];
+  __darwinAllowLocalNetworking = true;
```

Comment:
Why is this needed?

Reply:
It was needed for pypi but it's not needed anymore.